base/src/META-INF/blaze-base.xml

```diff
@@ -412,6 +412,9 @@
     <registryKey defaultValue="true"
                  description="By default Bazel plugin takes control and opens project if .ijwb folder is present. This property can be used to disable this behavior to allow to open .idea or other project models if they are exists."
                  key="bazel.project.auto.open.if.present"/>
```

Comment:
can't we just reuse this one and set it false by default?

Reply:
present is mutually exclusive of auto-import, so no